docs/cody/capabilities/proxy-setup.mdx

```diff
@@ -0,0 +1,136 @@
+# Proxy Setup
+
```

Comment:
```suggestion
# Running Cody via proxies

```

Something like this...? I don't know what the best title can be here. This almost feels better under the Installation section @MaedahBatool, and not as a 'Capabilities'

Reply:
We will be getting our Tabs component. Hopefully, next week, we can then move it under respective tabs. For now let's keep it under capabilities. I agree it's not a fully compatible place. :)